src/rust/madara/sources/mangaread/res/source.json

```
@@ -0,0 +1,18 @@
+{
+	"info": {
+		"id": "en.mangaread",
+		"lang": "en",
+		"name": "Manga Read",
```

I think Manga Read is used for https://mangaread.co/ in tachiyomi, so we should follow suit and call this MangaRead.org 
```suggestion
		"id": "en.mangareadorg",
		"lang": "en",
		"name": "MangaRead.org",
```